meta-mender-demo/recipes-core/psplash/psplash_git.bbappend

```diff
@@ -2,6 +2,17 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
 SRC_URI_append = " \
 	file://mender.io.png \
+	file://psplash-start.service \
+	file://psplash-quit.service \
 "
 
+inherit systemd
+
+SYSTEMD_SERVICE_${PN} = "psplash-start.service psplash-quit.service"
+
 SPLASH_IMAGES = "file://mender.io.png;outsuffix=default"
+
```

Comment:
Did a quick test on my Pi4, I had to add the following to "win" over meta-raspberrypi who set priority to 200.

```
ALTERNATIVE_PRIORITY = "300"
```

Reply:
Ah indeed, thanks for testing. I will update the recipe.